### PR TITLE
Migrate examples 2 E2E test to use KUTTL

### PR DESCRIPTION
**Pull request by iblancasa**
## Which problem is this PR solving?
- Migrate the E2E examples 2 tests to use KUTTL.

### CODE REVIEW

**Comment by rubenvp8510** [comment]
Is this ready for review?

File: Makefile [hunk 1]
```diff
@@ -542,7 +595,7 @@ prepare-e2e-kuttl-tests: build docker build-assert-job
 kuttl-e2e: prepare-e2e-kuttl-tests start-kind run-kuttl-e2e
 
 .PHONY: run-kuttl-e2e
-run-kuttl-e2e:
+run-kuttl-e2e: kafka
```
Comment (rubenvp8510) [inline Makefile:598]: I think this should be  part of test prepare target.

Comment (iblancasa) [reply Makefile:598]: Sure! I fixed this for #1598 but I didn't have the chance to apply the change here

File: tests/e2e/examples-simple-streaming/00-install.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Start Kafka
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: "pwd && cd ../../.. && make kafka"
```
Comment (rubenvp8510) [inline tests/e2e/examples-simple-streaming/00-install.yaml:5]: This test if fialing I think we need to change this to 

`  - script: " KAFKA_NAMESPACE=$NAMESPACE make kafka"`

**Review by rubenvp8510 — CHANGES_REQUESTED** [review]

**Review by rubenvp8510 — APPROVED** [review]

**Pull request merged**